.drone.yml

```diff
@@ -0,0 +1,90 @@
+pipeline:
+
+  docker-build:
+      image: docker:17.09.1
+      environment:
+        - DOCKER_HOST=tcp://172.17.0.1:2375
+      commands:
+        - docker build -t hocs-toolbox .
+      when:
+        branch: [master, refs/tags/*]
+        event: [push, tag]
+
+  install-docker-image:
+      image: docker:17.09.1
+      environment:
+        - DOCKER_HOST=tcp://172.17.0.1:2375
+      secrets:
+        - docker_password
+      commands:
+        - docker login -u=hocs-bot -p=$${DOCKER_PASSWORD} docker.digital.homeoffice.gov.uk
+        - docker tag hocs-toolbox docker.digital.homeoffice.gov.uk/hocs-toolbox:build-$${DRONE_BUILD_NUMBER}
+        - docker tag hocs-toolbox docker.digital.homeoffice.gov.uk/hocs-toolbox:latest
+        - docker push docker.digital.homeoffice.gov.uk/hocs-toolbox:build-$${DRONE_BUILD_NUMBER}
+        - docker push docker.digital.homeoffice.gov.uk/hocs-toolbox:latest
+      when:
+        branch: master
+        event: push
```

Comment:
I would switch this out for Quay.io, as this repo is public. It also means we can simplify this using a Drone plugin
```yaml
  - name: build & push
    image: plugins/docker
    settings:
      registry: quay.io
      repo: quay.io/ukhomeofficedigital/hocs-toolbox
      tags:
        - build_${DRONE_BUILD_NUMBER}
        - ${DRONE_COMMIT_SHA}
        - branch-${DRONE_COMMIT_BRANCH/\//_}
    environment:
      DOCKER_PASSWORD:
        from_secret: QUAY_ROBOT_TOKEN
      DOCKER_USERNAME: ukhomeofficedigital+hocs_quay_robot
```

Reply:
Done